### DISCUSSION TITLE
Hide and Show all children of ListBox prefab

### DISCUSSION THREAD

**Discussion by gregorysemah** [LanKuDot/game_modules#40] [General]
Hello and thanks for your work, it helps me a lot!!

Is there, by any simple way to show/hide content of a listbox prefab, based on the selected item in center of the scrollingList and with the OnListCenteredContentChanged method? Or just get the related listbox gameObject, in order to manipulate his content?

The goal is to show only children's content off the centered element and hide all the other children's content
Example: if there is a text element on the listBox, i want to show it only when it is centered with certainly OnListCenteredContentChanged kind of event

thanks in advance!

**Comment by LanKuDot** [comment]
Yes, you could use `OnCenteredBoxChanged` event to get the last and the current focused box and then toggle the content of them.
The type of passed box is `ListBox`, so you have to convert the type of it to the derived type to access the function for togging the content.

**Reply by gregorysemah** [reply]
Thanks four your quick answer, could you be more precise? there is no such event or example in your code,
how and where to handle this?

**Reply by LanKuDot** [reply]
Sorry, I got mixed up. That is the new feature of version 6.
It is completed, and I am making the documentation. I think it could be done in these days.
You could download the code in the master branch, if it is urgently needed.

**Reply by LanKuDot** [reply]
Hi! Version 6 is completed. You could download the new version from this repo or from the unity asset store few days later.
